deployments/warp_routes/TURTLE/ethereum-linea-deploy.yaml

```diff
@@ -0,0 +1,11 @@
+ethereum:
+  interchainSecurityModule: "0x0000000000000000000000000000000000000000"
```

Comment:
After removing the ism field in the monorepo pr, be sure to run the generate config script again. It should remove this field from here as well